cv32e40s/tests/programs/custom/riscv_arithmetic_basic_test_0/riscv_arithmetic_basic_test_0.S

```diff
@@ -11961,7 +11961,7 @@ mmode_intr_vector_15:
                   j mmode_intr_handler
                   1: j test_done
 
-.align 2
+.align 7
```

Comment:
7?!?   This is outside of my wheelhouse,  but why 7?

Reply:
This one puzzled me as I also initially thought this was byte alignment - It appears that risc-v is using size-log-2 alignment. log2(128) = 7: 

From
![image](https://user-images.githubusercontent.com/75469886/199485760-b7ae39c1-4d94-46b6-97fa-9c046a15dbdd.png)